The app keep on crashing after integration

**Describe the bug**
`
Process: com.example.app, PID: 8605
                                                                                                    java.lang.ExceptionInInitializerError
                                                                                                    	at com.freeraspreactnative.FreeraspThreatHandler.onAllChecksFinished(FreeraspThreatHandler.kt:93)
                                                                                                    	at com.aheaditec.talsec_security.security.api.ThreatListener.onReceive(Unknown Source:667)
                                                                                                    	at androidx.localbroadcastmanager.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:319)
                                                                                                    	at androidx.localbroadcastmanager.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:127)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:222)
                                                                                                    	at android.os.Looper.loop(Looper.java:314)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8857)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:565)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013)
                                                                                                    Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int com.freeraspreactnative.events.RaspExecutionStateEvent$AllChecksFinished.getValue()' on a null object reference
                                                                                                    	at com.freeraspreactnative.events.RaspExecutionStateEvent.<clinit>(RaspExecutionStateEvent.kt:18)
                                                                                                    	at com.freeraspreactnative.FreeraspThreatHandler.onAllChecksFinished(FreeraspThreatHandler.kt:93) 
                                                                                                    	at com.aheaditec.talsec_security.security.api.ThreatListener.onReceive(Unknown Source:667) 
                                                                                                    	at androidx.localbroadcastmanager.content.LocalBroadcastManager.executePendingBroadcasts(LocalBroadcastManager.java:319) 
                                                                                                    	at androidx.localbroadcastmanager.content.LocalBroadcastManager$1.handleMessage(LocalBroadcastManager.java:127) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106) 
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:222) 
                                                                                                    	at android.os.Looper.loop(Looper.java:314) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8857) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:565) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1013) 
`

**To Reproduce**
Add to an expo 53 project

**Expected behavior**
Wont't crash

**Screenshots**
No

**Please complete the following information:**
 - Device: Android AVD
 - OS version: SDK36
 - Version of freeRASP: 4.3.1

**Additional context**
No

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.3.2] - 2025-01-03
+
+- Android SDK version: 17.0.1
+- iOS SDK version: 6.13.0
+
+### React Native
+
+#### Fixed
+
+- Resolved potential NullPointerException when execution state events are being sent 
+
 ## [4.3.1] - 2025-12-16
 
 - Android SDK version: 17.0.1

--- a/android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt
+++ b/android/src/main/java/com/freeraspreactnative/FreeraspReactNativeModule.kt
@@ -59,6 +59,7 @@ class FreeraspReactNativeModule(private val reactContext: ReactApplicationContex
   init {
     appReactContext = reactContext
     reactContext.addLifecycleEventListener(lifecycleListener)
+    initializeEventKeys()
   }
 
   @ReactMethod
@@ -87,6 +88,12 @@ class FreeraspReactNativeModule(private val reactContext: ReactApplicationContex
     } catch (e: Exception) {
       promise.reject("TalsecInitializationError", e.message, e)
     }
+  }
+
+  // Trigger lazy initialization of the freeRASP events
+  private fun initializeEventKeys() {
+    ThreatEvent.ALL_EVENTS
+    RaspExecutionStateEvent.ALL_EVENTS
   }
 
   /**

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "freerasp-react-native",
-  "version": "4.3.1",
+  "version": "4.3.2",
   "description": "React Native plugin for improving app security and threat monitoring on Android and iOS mobile devices.",
   "main": "lib/commonjs/index",
   "module": "lib/module/index",

